pkg/controller/cache.go

```diff
@@ -69,9 +68,34 @@ func (s *InMemoryStore) GetByPvcUID(pvcUID types.UID) (*TopologyInfo, error) {
 	defer s.mutex.RUnlock()
 	info, found := s.data[pvcUID]
 	if !found {
-		return nil, fmt.Errorf("topology object with pvcUID '%s' not found", pvcUID)
+		// To prevent printing UID in logs, we return a standard error
+		return nil, fmt.Errorf("topology object with given pvcUID not found")
```

Comment:
what's wrong with printing pvcUID? Also I thought we are returning nil error when the entry is not found so that we can distinguish between an error and not found?